nixos/modules/services/x11/urxvtd.nix

```diff
@@ -13,8 +13,7 @@ in {
     default = false;
     example = true;
     description = ''
-      Enable urxvtd, the urxvt terminal daemon. To use urxvtd, first
-      "export RXVT_SOCKET=/run/user/$(id -u)/urxvtd-socket" then run
+      Enable urxvtd, the urxvt terminal daemon. To use urxvtd, run
       "urxvtc".
```

Comment:
ok just noticed that systemd user services are not documented at all there is only one hint in the emacs module: https://nixos.org/nixos/manual/index.html#idm140737315633488